nss-day-cohort-39/holiday-road-cosmic-chameleons
# added parks select dropdown rendered through json file

### CHANGES

- Added containers to index to hold components in Itinerary Preview
 - Ran JSON server to house copy of data from parks api
- Got dropdown box working for park selector

### TESTING

- Run json file "json-server -p 8090 -w parks.json" from api folder
- Check that dropdown functions and displays list of parks
- Check that new tags are in accordance with team-discussed naming rules
